### DISCUSSION TITLE
Scaffolding admin controllers and views

### DISCUSSION THREAD

**Discussion by nikoma** [bdavidxyz/bootrails_place#2] [General]
Is there a way to scaffold admin controllers and views like your articles_controller? I like the way how you created backend admin controllers and views as well as frontend controllers (WordPress style). However, is there a scaffolder for this? Otherwise, would active_admin not be faster to implement?

**Comment by bdavidxyz** [comment]
This is a good question. I'm writing documentation about this. We have tried active_admin, we have tried administrate, but finally, over the long-term, this is not a good idea. Short-term speaking, administrate is probably the best tool out there. But soon or late, you will fight the admin gem, because your need will not match the way the gem was designed. The best way is to start from Rails scaffolders, or your own code, and customise them accordingly. So the answer is : copy/paste existing views and controllers, and modify code according to your needs.
